lib/Conversion/ImportVerilog/Expressions.cpp

```diff
@@ -53,6 +53,13 @@ struct ExprVisitor {
     return {};
   }
 
+  // Handle references to the left-hand side of a parent assignment.
+  Value visit(const slang::ast::LValueReferenceExpression &expr) {
```

Comment:
Do continuous assignments use a different system for lvalues or can they not be nested?

Reply:
They don't allow for the `+=` syntax. `+=` is always a blocking assignment within a procedural block.